index.html

```diff
@@ -0,0 +1,238 @@
+                    <li><a href="./about.html">Sponsor</a></li>
+                    <li><a href="./about.html">News</a></li>
+                    <li><a href="./about.html">PANACCS Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
+        <div class="container">
+            <div class="header-nav-phone">
+                <button type="button" class="menu-btn">
+                    <img src="./Images/Union.svg" alt="menu link">
+                </button>
+                <nav class="header-nav-mobile hidden">
+                    <button type="button" class="btn-close-nav">
+                        <img src="./Images/close_icon.svg" alt="cancel icon">
+                    </button>
+                    <ul>
+                        <li>
+                            <a href="./about.html" class="link-about">
+                                About
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-program">
+                                Program
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-join">
+                                Join
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-sponsor">
+                                Sponsor
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-news">
+                                News
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="link-cc-campaign">
+                                PANACCS Campaign
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="headline">
+                <h3>
+                    "Hello! African Summit!"
+                </h3>
+                <h1>
+                    PAN African Cloud Computing Summit 2022
+                </h1>
+                <p class="headline-paragraph-description">
+                    An impactful event  round around  creating a positive change with speakers 
+                    from over 54 African nations and world beyond is talking place by October, in Nigeria.
+                </p>
+                <h2 class="headline-date">
+                    2022.09.14(WED) ~ 15(FRI)
+                </h2>
+                <p class="headline-paragraph-location">
+                    @International Conference Center, Abuja, Nigeria.
+                </p>
+            </div>
+        </div>
+    </header>
+    <main>
+        <div class="container">
+            <div class="main-title">
+                <p>Main Program</p>
+            </div>
```

Comment:
- [ ] For good semantics tags, kindly make sure you use on all titles of sections a  `hx`(**h2, h3,...**) tag instead of the `p` tag.
 Note: There should be at least 1, but no more than 1, `<h1>` tag on each page for the page title(e.g.: **PAN African Cloud Computing Summit 2022** )